gulp-file.ts

```diff
@@ -109,18 +121,19 @@ gulp.task('min', ['concat'], () => {
         .pipe(gulp.dest(paths.src.dist));
 });
 
-// This task copies the HTML and library js into a dist folder.
-// TODO: update to build and min the site js.
+// This task copies the HTML, angular js and library js into a dist folder.
 gulp.task('app:build', ['min'], () => {
-    return gulp.src([files.site.html, paths.src.dist + wildcard])
+    gulp.src([files.site.html, paths.src.dist + wildcard])
         .pipe(gulp.dest(paths.site.app));
 });
 
 gulp.task('app:serve', ['app:build'], () => {
-    let gls = require('gulp-live-server');
+    let server = createServer();
+    server.start();
+});
 
-    let portNumber = 8080;
-    let server = gls.static(paths.site.app, portNumber);
+gulp.task('app:connect', [], () => {
+    let server = createServer();
     server.start();
```

Comment:
Actually. given that function is being used twice and both times the server is being started maybe the `.start()` portion belongs in `createServer`, as a `startServer()` method. What do you think?